src/utils/addresses.ts

```diff
@@ -2,7 +2,14 @@ const getPartiallyHiddenEthereumAddress = (ethereumAddress: string): string => {
   const firstAddressSlice = ethereumAddress.slice(0, 6);
   const secondAddressSlice = ethereumAddress.slice(ethereumAddress.length - 4);
 
-  return `${firstAddressSlice}･･･${secondAddressSlice}`;
+  return `${firstAddressSlice} ･･･ ${secondAddressSlice}`;
 };
 
-export { getPartiallyHiddenEthereumAddress };
+const getShortEthereumAddress = (ethereumAddress: string): string => {
```

Comment:
I think that the correct name for the function should be `getShortedEthereumAddress` right?

Reply:
I don't think `getShortEthereumAddress` is incorrect, but yeah I guess we can also use the participle `getShortedEthereumAddress`. Although `getShortenedEthereumAddress` sounds even better to me.

Reply:
![image](https://user-images.githubusercontent.com/15896177/183852063-dbda9519-317f-44ea-846c-76c56b95bfe5.png)

Reply:
Done.